mods/ra/maps/allies-10a/allies10a.lua

```diff
@@ -120,13 +120,13 @@ LaunchMissiles = function()
 			-- Completing immediately indicates that the FMV is not available
 			-- Fall back to a text message
 			if fmvStart == DateTime.GameTime then
-				Media.DisplayMessage("Commander, we're tracking four missiles. They must be deactivated! We are scrambling a team to assult the missile control bunker. Clear the way and capture the enemy command center. Hurry!", "LANDCOM 16")
+				Media.DisplayMessage(UserInterface.Translate("deactivate-four-missiles"), "LANDCOM 16")
```

Comment:
We translate `Tanya` but not `LANDCOM 16`? (I don't know if it is necessary, just an observation.)

Reply:
Actually there is a translation for it used in a different mission, so this should probably change.

Reply:
Other languages may use different letters so it definitely should get a translation

Reply:
Tanya might be Tanja in Germany and after thinking about it, I just made everything translatable because who knows how this gets translated in say Chinese.